server-phoenix/lib/helios_web/schema/queries/event_collection.ex

```diff
@@ -6,7 +6,7 @@ defmodule HeliosWeb.Schema.Queries.EventCollection do
   object :event_collection_queries do
     field :events, :event_collection do
       description("MojoTech slack/github events")
-      arg(:created_after, :datetime)
+      arg(:created_after, :string)
```

Comment:
nit: could we rename this arg `:after` to match the ruby version? then we could drop all the client/ruby changes?

Reply:
I should've noted it. `after` is a reserved word in elixir so that's why I chose to go with `created_after`